server/views/pages/contact-us.ejs

```diff
@@ -20,7 +20,7 @@
               <span class="govuk-body" style="white-space: nowrap;">0203 788 4636</span>.
             </p>
             <p class="govuk-body">
-              Alternatively, you can <a href="mailto:checkmydiary@digital.justice.gov.uk"  class="govuk-link">email us</a>.
+              Alternatively, you can <a href="mailto:checkmydiary@digital.justice.gov.uk"  class="govuk-link">email us at checkmydiary@digital.justice.gov.uk</a>.
```

Comment:
😬  bit late now, but can we substitute both of these with a variable? I'm guessing this is also just hardcoded in the properties file isn't it

Reply:
Interestingly I did at first, but then spotted its used elsewhere too and decided against. I think raise a ticket to consolidate the whole lot.